src/main.js

```diff
@@ -115,14 +123,14 @@ function getRandomIndex(array) {
 }
 
 function createRandomPoster() {
-  var imageURL =  images[getRandomIndex(images)];
+  var imageURL = images[getRandomIndex(images)];
   var title = titles[getRandomIndex(titles)];
   var quote = quotes[getRandomIndex(quotes)];
 
-  return currentPoster = new Poster(imageURL, title, quote);
+  currentPoster = new Poster(imageURL, title, quote);
 }
 
-var randomPoster = createRandomPoster();
+createRandomPoster();
```

Comment:
Since we didn't end up using this randomPoster variable, it was a good idea to get rid of it.